testing
testWithErrorInSetup
	self assert: true